libbeat/template/load_integration_test.go

```diff
@@ -60,55 +61,227 @@ func defaultESLoader(t *testing.T) *ESLoader {
 	return NewESLoader(client)
 }
 
-func TestCheckTemplate(t *testing.T) {
-	loader := defaultESLoader(t)
+func testFields(t *testing.T, name string) []byte {
+	file, err := filepath.Abs(filepath.Join("testdata", "default_fields.yml"))
+	require.NoError(t, err)
+	fields, err := ioutil.ReadFile(file)
+	require.NoError(t, err)
+	return fields
+}
+
+func TestESLoader_Load(t *testing.T) {
+	// Setup ES
+	loader := testEsLoader(t)
+	client := loader.client
+
+	name := "testbeat"
```

Comment:
we're back at the resource naming discussion. If some test fails, then we loose context cause the next test will remove the state of the current test. Stateful tests seting up resources should be isolated by having distinct resource names.